sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb

```diff
@@ -4,16 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>Present tense must be used: "an IAM role that will allow it" -&gt; "an IAM role that allows it"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>